frontend/src/App.js

```diff
@@ -206,6 +209,22 @@ function App() {
               <SavedJobs />
             </Protection>
           }
+        />
```

Comment:
The redirection to the settings page is routed correctly from the Navigation bar.